src/azure-stac-cli/stac/common/__utilities.py

```diff
@@ -18,18 +18,14 @@ def convert_json_to_ndjson(file_path: str):
     
     ndjson_name = Path(file_path).stem + ".ndjson"
     
-    file_content = ''
-
     try:
         with open(file_path, "r") as file:
             
             data = json.load(file)
             
-            file_content = json.dumps(data, separators=(',',':'))
-
         with open(ndjson_name, 'w') as outfile:
             
-            outfile.write(f'{file_content}\n')
+            json.dump(data, outfile, separators=(',',':'))
```

Comment:
It could be the way GH is showing the diff, but it looks like indentation for this is incorrect. The `with open(ndjson_name, 'w') as outfile:` block should be indented since `data` is defined in the block above.

Reply:
Good catch. Also, I haven't tested some changes. It would be a single end to end test at the end after addressing all the feedback. Thanks for catching this. 
In the meantime, fixed this missing indentation.